.github/workflows/flatpak.yml

```diff
@@ -18,18 +18,41 @@ env:
   YOUTUBE_SECRET_HASH: ${{ secrets.YOUTUBE_SECRET_HASH }}
 
 jobs:
+  check_tag:
+    name: Check release tag
+    runs-on: [ubuntu-latest]
+    outputs:
+      valid_tag: ${{ steps.check_tag.outputs.valid_tag }}
+      matrix: ${{ steps.check_tag.outputs.matrix }}
+    steps:
+      - name: Check the tag
+        id: check_tag
+        run: |
+          shopt -s extglob
+
+          case ${GITHUB_REF##*/} in
+            +([0-9]).+([0-9]).+([0-9])?(-beta|-rc) ) echo "::set-output name=valid_tag::${{ toJSON(true) }}" ;;
```

Comment:
It would suffice, while I made it I just wanted to avoid to put two times `::set-output name=valid_tag::${{ toJSON(true) }}` and separate the set up of each output.

Reply:
Made all the asked changes.